WalletWasabi.Documentation/Guides/HardwareWalletIntegration.md

```diff
@@ -0,0 +1,17 @@
+Wasabi's main priorities regarding hardware wallets:
+- Compatibility
+- Privacy
+
+1. Integrate your hardware wallet into HWI https://github.com/bitcoin-core/HWI
+2. Add some tests into HWI, so compatibility problems can be caught at early stages
+3. Test the compatibility with Wasabi. Import, Receive, Send, Recover.
+4. Write some Kata tests (manual tests), that can be used to detect compatibility issues when: Wasabi release, new firmware, etc. https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Tests/AcceptanceTests/HwiKatas.cs
```

Comment:
What is Kata test? Is that a Wasabi thing? Is that some general way of testing?

Reply:
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Tests/AcceptanceTests/HwiKatas.cs#L16-L22